gulpfile.js

```diff
@@ -24,6 +24,15 @@ const config = {
 	// Path to the default configuration file for bundler.
 	BUNDLE_DEFAULT_CONFIG: 'dev/bundles/build-config-standard.js',
 
+	DOCUMENTATION: {
+		// Path to the built editors for samples.
+		EDITORS_DIST: '.docs/assets/scripts/samples',
```

Comment:
OK, I can see that you meant to use `.docs`. Then this is bad. You should use some directory inside `build/docs/`. Also, aren't those files needed only temporarily?